packages/backend/src/services/redis-storage.ts

```diff
@@ -0,0 +1,58 @@
+﻿import type { createClient } from 'redis';
+
+export type RedisClientType = ReturnType<typeof createClient>;
+
+export class RedisStorageService {
+  private client: RedisClientType;
+
+  constructor(redisClient: RedisClientType) {
+    this.client = redisClient;
```

Comment:
You don't need to pass redisClient. Just instantiate it inside redisStorageService